packages/webapp/src/views/credits-purchase/credits-terms-and-conditions.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'form-layout.html' %}
+
+{% set serviceName = "Purchase statutory biodiversity credits" %}
+{% set pageHeading = "Terms and Conditions for the purchase of Statutory Biodiversity Credits" %}
+
+{% block formContent %}
+    <div class="govuk-body">
+        {% if err %}
+            {% include "errorbanner.html" %}
+        {% endif %}
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+              <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
+                <p>The statutory biodiversity credit scheme is run by Natural England on behalf of the Department for Environment, Food and Rural Affairs (Defra) as part of the Environment Act 2021.</p>
+                
+                <h2 class="govuk-heading-m">Privacy notice</h2>
+                <p class="govuk-body">Our <a href="#">privacy notice </a>sets out how we use the personal data.</p>  
```

Comment:
Use the same link as we use in the footer to the privacy notice: https://www.gov.uk/government/publications/natural-england-privacy-notices/biodiversity-net-gain-privacy-notice